### PR TITLE
fail if no sampler output is found for the poller

### PR DESCRIPTION
**Pull request by deads2k**
builds on https://github.com/openshift/origin/pull/28189

### CODE REVIEW

File: pkg/monitortests/network/disruptionpodnetwork/host-network-to-host-network-poller-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        - command:
+            - /usr/bin/openshift-tests
+            - disruption
+            - watch-endpoint-slice
+            - --output-file=/var/log/persistent-logs/disruption-host-to-host.jsonl
+            - --disruption-backend-prefix=host-to-host
+            - --disruption-target-service-name=host-network-service
+            - --stop-configmap=stop-collecting
+            - --my-node-name=$(MY_NODE_NAME)
+          image: docker.io/deads2k/openshift-tests:latest
```
Comment (neisw) [inline pkg/monitortests/network/disruptionpodnetwork/host-network-to-host-network-poller-deployment.yaml:33]: Nit: I know these get replaced but wondering if there is value documenting that here like you did in other prs with something like `docker.io/to-be-replaced/openshift-tests:latest`

File: pkg/monitortests/network/disruptionpodnetwork/monitortest.go [hunk 2]
```diff
@@ -142,6 +173,16 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 		return err
 	}
```
Comment (neisw) [inline pkg/monitortests/network/disruptionpodnetwork/monitortest.go:175]: Does the order of execution matter at all, any value to deploying the target and service before the pollers?

Comment (deads2k) [reply pkg/monitortests/network/disruptionpodnetwork/monitortest.go:175]: > Does the order of execution matter at all, any value to deploying the target and service before the pollers?

It does not.  The targets don't look at anything.  The pollers use the information about what is live to decide what to sample.

File: pkg/monitortests/network/disruptionpodnetwork/host-network-to-host-network-poller-deployment.yaml (hunk 1 above)
Comment (deads2k) [reply pkg/monitortests/network/disruptionpodnetwork/host-network-to-host-network-poller-deployment.yaml:33]: > Nit: I know these get replaced but wondering if there is value documenting that here like you did in other prs with something like `docker.io/to-be-replaced/openshift-tests:latest`

Promised as an IOU

**Comment by deads2k** [comment]
transitive lgtm via https://github.com/openshift/origin/pull/28191

**Comment by deads2k** [comment]
no conflict with the PR that merged

**Pull request merged**